### DISCUSSION TITLE
Custom Theme Type Warning

### DISCUSSION THREAD

**Discussion by haibert** [GeekyAnts/NativeBase#4374] [Q&A]
So I’m creating a custom theme object for NativeBase and inside of it I am adding some extra properties like “backgroundColor”. Later when accessing these properties through useTheme() the code works, but I get a type warning. Is there a way to silence them? In reality the prop is there and my code works absolutely fine.. its only a type warning.

**Comment by sanna231** [comment]
Hey @haibert , Seems like it should be in issues. We've moved it to issues. You can track [here](https://github.com/GeekyAnts/NativeBase/issues/5224). Thanks for reporting it.
